src/handlers/search-runner.js

```diff
@@ -92,31 +92,35 @@ const constructSearchContext = async (event) => {
     searchContext.from = (page - 1) * size;
   }
 
-  // if (queryStringParameters.sort) {
-  //   //TODO
-  // }
-
   return searchContext;
 };
 
-const fromQueryString = ({ queryStringParameters, requestContext }) => {
-  if (queryStringParameters?.query) {
-    return {
-      query: {
-        query_string: {
-          query: queryStringParameters.query,
-        },
-      },
-    };
-  } else {
-    return {
-      query: {
-        query_string: {
-          query: "*",
-        },
+const fromQueryString = ({ queryStringParameters, _requestContext }) => {
+  let request = {
+    query: {
+      query_string: {
+        query: queryStringParameters?.query || "*",
       },
-    };
+    },
+  };
+
+  if (queryStringParameters?.sort)
+    request["sort"] = parseSortParameter(queryStringParameters.sort);
```

Comment:
Can we move this statement up into `constructSearchContext` near the size/from/page setters so that other `GET` routes that pre-fill the request body (e.g., `/works/{id}/similar` and `/collections`) can also take advantage of `sort`?